Docker/test/curl/test_curl.sh

```diff
@@ -1,36 +1,25 @@
-#!/bin/bash
+#!/bin/sh
 
-# Wikibase test
-if curl --fail  --retry 5 --retry-all-errors --retry-delay 10 --max-time 10 --retry-max-time 60 --show-error --output /dev/null --silent "$WIKIBASE_SERVER"/wiki/Main_Page; then
-    echo 'Successfully loaded the wiki main page!'
-else
-    echo "Could not retrieve main page"
-    exit 1
-fi
+check_if_up() {
+    URL=$1
+    PATH=$2
 
-# QueryService test blazegraph root page
-if curl --fail --show-error --output /dev/null --silent "$QUERYSERVICE_SERVER"/bigdata/namespace/wdq/sparql; then
-    echo 'Successfully loaded the QueryService page!'
-else
-    echo "Could not retrieve root page"
-    exit 1
-fi
+    FULL_URL="$1$2"
 
-# QueryService UI test
-if curl --fail --show-error --output /dev/null --silent "$QUERYSERVICE_UI_SERVER"/; then
-    echo 'Successfully loaded the QueryService UI page!'
-else
-    echo "Could not retrieve UI root page"
-    exit 1
-fi
+    if [ -z "$URL" ]; then
+        exit 0
```

Comment:
This seems like it should be a `return` rather… if `$URL` is empty, this script will exit successfully.